spec/menu_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'menu'
+
+describe Menu do
+  let(:dish) { double(:dish, name: 'paneer_tikka_masala', price: 10) }
+  let(:second_dish) { double(:second_dish, name: 'pizza', price: 20) }
+
+  before do
+    subject.menu = []
+  end
+
+  it 'can respond to add dish' do
+    expect(subject).to respond_to(:add_dish)
+  end
+
+  it 'can add a named dish to the menu' do
+    allow(Dish).to receive(:new).and_return(dish)
+    subject.add_dish('paneer_tikka_masala', 10)
+    expect(subject.menu).to eq([dish])
+  end
+
+  it 'can remove a specific dish from the menu' do
+    allow(Dish).to receive(:new).and_return(dish)
```

Comment:
Same remark as for the other class, if you were to inject the different dish instances into `Menu.new` (right now, they're hardcoded in `Menu#initialize`), you wouldn't have to stub `Dish.new` and could inject doubles in this test, in place of the concrete `Menu` instances